<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistrationWindow</class>
 <widget class="QDialog" name="RegistrationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>231</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Авторизация</string>
      </property>
      <widget class="QWidget" name="layoutWidget_2">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>42</y>
         <width>193</width>
         <height>175</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="login_label">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Логин</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="name_label">
              <property name="text">
               <string>Имя</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="pass1_label">
              <property name="text">
               <string>Пароль</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="pass2_label">
              <property name="text">
               <string>Пароль</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLineEdit" name="login_lineEdit"/>
            </item>
            <item>
             <widget class="QLineEdit" name="name_lineEdit"/>
            </item>
            <item>
             <widget class="QLineEdit" name="pass1_lineEdit"/>
            </item>
            <item>
             <widget class="QLineEdit" name="pass2_lineEdit"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="makeAcc_Button">
          <property name="text">
           <string>Создать аккаунт</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="back_Button">
      <property name="text">
       <string>Назад</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
